components/core/CreateShape.js

```diff
@@ -247,21 +247,24 @@ const CreateShape = (props) => {
 
         if (clipPathType === "polygon") {
             let formulaNumbers = formula.slice(formula.indexOf("(") + 1, formula.indexOf(")"));
-            formulaNumbers = formulaNumbers.split(", "); 
+            formulaNumbers = formulaNumbers.split(","); 
             newVerticeCoordinates = formulaNumbers.map(x => {
-            let percentageArray = x.split(" ");
-            return {
-                "x": percentageArray[0], 
-                "y": percentageArray[1],
-            }
+                let values = x.trim();
+                let xValue = values.substring(0, values.indexOf("%") + 1).trim();
```

Comment:
@williamzhu17, the clip-path value can take other units than %. We should support them in our logic.

Reply:
they should now be able to take in px values